marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.tsx

```diff
@@ -130,26 +130,10 @@ const ZeroStateExplorer = (props) => {
               <Card className={styles.largeCard} bordered={false}>
                 <Row>
                   <Col span={24}>
-                    <div className={styles.input}>
-                      <Input
-                        style={{width: 700}}
-                        placeholder="Enter text to search for"
-                        addonBefore={entityMenu}
-                        onChange={onChange}
-                        allowClear
-                        data-cy="search-bar"
-                        data-testid="search-bar"
-                      />
-                    </div>
-                  </Col>
-                </Row>
-                <Row>
-                  <br />
-                  <Col span={12}>
                     <div className={styles.database}>
                       <p className={styles.databaseLabel}>Database:</p>
                       <MLRadio.MLGroup
-                        style={{}}
+                        style={{width: "160px"}}
```

Comment:
Same comment here as well, let's see if you can use className attribute here and place the css to the modiule.scss file.

Reply:
pushed commit.